projects/packages/my-jetpack/src/class-jetpack-manage.php

```diff
@@ -90,14 +121,15 @@ public static function could_use_jp_manage( $min_sites = 2 ) {
 	 *
 	 * @return bool Return true if the user is a partner/agency, otherwise false.
 	 */
-	public static function is_agency_account() {
+	private static function is_agency_account() {
 		// Only proceed if the user is connected to WordPress.com.
 		if ( ! ( new Connection_Manager() )->is_user_connected() ) {
 			return false;
 		}
 
 		// Get the cached partner data.
-		$partner = get_transient( 'jetpack_partner_data' );
+		$partner = false;
```

Comment:
This looks ok to me, but I wanted to check if the change is intentional.

Reply:
No this was not intentional, it was for testing 😅 Thank you for catching this